latex2sympy.py

```diff
@@ -656,7 +656,7 @@ def convert_func(func):
         else:
             args = func.func_multi_arg_noparens().split(",")
 
-        args = list(map(process_sympy, args))
+        args = list(map(lambda arg: process_sympy(arg, VARIABLE_VALUES), args))
```

Comment:
@tirkarp ah that makes a lot of sense. nice fix!

Reply:
@builtbywill Is that an approval? 😄

Reply:
@gavin-brown woops, totally forgot to approve